### DISCUSSION TITLE
【请教Web端】numVideos 获取到视频数个数以后，怎么替换掉这个视频

### DISCUSSION THREAD

**Discussion by leohandsome1688** [Tencent/libpag#2036] [Q&A] [answered]
利用numVideos 获取到视频数个数以后，我现在想替换掉其中的一个视频，但是不知道怎么操作去替换？我用获取图片的方式去处理，但是不成功~~有知道的大神，可以指点一下吗？跪谢~

**Comment by kevingpqi123** [comment, marked as answer]
numVideos 指的是素材中含有 BMP 预合成的数量，这是素材的一部分，是不能替换的

<img width="398" alt="image" src="https://github.com/Tencent/libpag/assets/16609221/318ad14f-bbd8-47c7-8c94-9b496396d049">


这里可以了解下 BMP 预合成：https://pag.art/docs/ae-bmp-guide.html
